sonic_platform_base/sonic_xcvr/api/xcvr_api.py

```diff
@@ -283,3 +291,46 @@ def set_power_override(self, power_override, power_set):
             False if not
         """
         raise NotImplementedError
+
+    def get_paging(self):
+        """
+        Retrieves the paging status of this xcvr, i.e. whether the memory map is paged
+
+        Returns:
+            A Boolean, True if paging is implemented, False otherwise (i.e. flat memory)
+        """
+        raise NotImplementedError
+
+    def get_tx_power_support(self):
```

Comment:
The intention is for this to act as a helper method in our testing so that we can adjust our expectation in the test case based on whether the feature is actually supported in the module or not, e.g. if this method returns False then get_tx_power() should return all 'N/A's.